test_start_stop_ioc_through_proc_control.py

```diff
@@ -0,0 +1,69 @@
+import os
+import unittest
+from utilities.utilities import g
+from six.moves import range
+import requests
+import time
+
+
+class TestPlotting(unittest.TestCase):
```

Comment:
These tests appear to be totally unrelated to the ticket. Instead, they are an exact copy-paste of the tests in `test_plotting.py`.

Can you recheck your git commits?